ext/drreg/drreg.c

```diff
@@ -105,13 +137,28 @@ typedef struct _reg_info_t {
 
 #define GPR_IDX(reg) ((reg)-DR_REG_START_GPR)
 
+/* ZMM registers are used to manage SIMD data, resizing to smaller regs when needed */
```

Comment:
If I was new to the code I wouldn't know what this means. If anything, we resize the smaller ones to zmm right? Maybe say that we use the zmm register range to search and iterate over simd registers? I would just remove the comment though.